test/service/test_response.py

```diff
@@ -1417,6 +1417,44 @@ def test_ordering(self):
             self.assertEqual(hit_sort_values['asc'], sorted(hit_sort_values['asc']))
             self.assertEqual(hit_sort_values['desc'], sorted(hit_sort_values['desc'], reverse=True))
 
+    def test_missing_field_sorting(self):
+        """
+        Test that sorting by a field that doesn't exist in all hits produces
+        results with the hits missing the field placed at the end of a
+        ascending sort and the beginning of a descending sort.
+        """
+        ascending_values = [
+            ['induced pluripotent'],
+            ['induced pluripotent'],
+            ['primary', 'stem cell-derived'],
+            None,  # The last 4 hits don't have any 'cellLines' inner entities
+            None,  # so for purposes of this test we use None to represent
+            None,  # that there is a hit however it has no 'cellLineType'.
```

Comment:
```suggestion
            None,  # that there is a hit but that it has no 'cellLineType'.
```

Unless your form is grammatically correct. Not sure though.